### ISSUE TITLE
Add Option (Toggle) to Delete After Conversion Operation

### ISSUE THREAD

**Issue by OkyDooky** [deckerst/aves#867]
Thanks to the app's ability to preview media durong the selection process, I sometimes find myself selecting in a "staggard" way, as opposed to a monolithic block, and then struggle with remembering exactly which ones I did and did not select for conversion from a batch.
For example, today, I converted 152 images, but when selecting all the ones I remembered, it came to 153 to delete. Unless I stick to just doing smaller batches (which kind of defeats the benefit of the peviewing during selecting feature), I have no consistently convenient way to compare the selections (to convert vs to delete).

Basically, I'm requesting a toggle like the "Write Metadata" one in the image conversion dialogue that has Aves move the selected images to the recycle bin after a successful conversion operation.

**Comment by deckerst** [comment]
I won't do that, because adding such a toggle would be risky and likely to result in data loss if something goes wrong with the first step of the process.

In your case, you could tag all your selected items. Then you can easily identify all your tagged items to convert them, then delete them.

**Comment by OkyDooky** [comment]
>likely to result in data loss

That's why I specified "moving to the recycle bin" instead of outright deleting. Would that still be an issue, on some technical level?

I appreciate the other suggestion. It is less convenient, but can get the job done.

**Comment by deckerst** [comment]
No technical difficulty. I just don't think it's a good addition, especially because it would be for a niche need that has a workaround.
